src/views/News/index.js

```diff
@@ -188,86 +472,106 @@ export default class StudentNews extends Component {
   }
 
   async handleSubmit() {
-    // create the JSON newsItem object to post
-    let newsItem = {
-      categoryID: this.state.newPostCategory,
-      Subject: this.state.newPostSubject,
-      Body: this.state.newPostBody,
-    };
+    let newImage;
 
-    // submit the news item and give feedback
-    let result = await newsService.submitStudentNews(newsItem);
-    if (result === undefined) {
-      this.updateSnackbar('News Posting Failed to Submit');
-    } else {
-      this.updateSnackbar('News Posting Submitted Successfully');
+    if (this.state.showCropper != null) {
```

Comment:
You should use the Strict Inequality operator `!==` as a general principle in JavaScript. The non-strict version `!=` can behave in unexpected ways and cause frustrating bugs.